tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py

```diff
@@ -2260,3 +2254,15 @@ def column_transformer_pipe(task_id):
         run2 = column_transformer_pipe(23)  # only numeric
         TestBase._mark_entity_for_removal("run", run2.run_id)
         self.assertEqual(run1.setup_id, run2.setup_id)
+
+    def test_for_refit_time_in_basesearchCV(self):
```

Comment:
Could you please explain the difference to the test above?

Reply:
This branch was initially created after checking out from the earlier PR we did which added the refit times. This unit test was removed from there as you pointed out this unit test is testing for scikit-learn. We should instead incorporate this check into an existing unit test, which I resulted in [this unit test](https://github.com/openml/openml-python/blob/fix_895/tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py#L1248).
I did rebase this branch but this function remained.

I'll remove it in the next push.